src/os/ObjectStore.h

```diff
@@ -393,7 +393,10 @@ class ObjectStore {
       __le32 oid;
       __le64 off;
       __le64 len;
-      __le32 dest_cid;
+      union {
+	__le32 fadvise_flags; 		//OP_WRITE
+	__le32 dest_cid;
+      };
       __le32 dest_oid;                  //OP_CLONE, OP_CLONERANGE
```

Comment:
If so, I think maybe we should refact the whole Op to union. It is really strange now :-P.